components/appbar/overview.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Overview
+page_title: AppBar Overview
+description: Overview of the AppBar component for Blazor.
+slug: appbar-overview
+tags: telerik,blazor,appbar,navbar
+published: True
+position: 0
+---
+
+# Blazor AppBar Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/appbar" target="_blank">Blazor AppBar component</a> helps you build navigation bars for your application seemingly. This article explains the available features.
+
+## Creating Blazor AppBar
+
+1. Add the `<TelerikAppBar>` tag to a Razor file.
+1. Use the `<AppBarSection>` child tag to add content to the AppBar component.
+1. (optional) Use [spacers or separators](#content-dividers) to add visual distinction between the sections in the AppBar.
+1. (optional) Set the [position](#positioning) of the AppBar.
+
+>caption Basic configuration of the Telerik AppBar
+
+````CSHTML
+<TelerikAppBar>
+    <AppBarSection>
+        <span>Our Logo</span>
+    </AppBarSection>
+
+    <AppBarSpacer Size="25%"></AppBarSpacer>
```

Comment:
The different ways to set `Size` make the empty spaces look uneven and the layout not so good. Better leave these details for the dedicated Separators article, and make the first impression better.

Also, consider `<AppBarSeparator />` between the three text items in the middle, which is more real-world too.